src/render/pixie/render_help.py

```
@@ -1,84 +1,243 @@
+    'tetris': '俄罗斯方块',
+    'guess-interval': '区间猜数',
+    'guess-1a2b': '1A2B',
+    'git-cmd': 'Git 管理',
+    'misc1': '趣味工具',
+    'misc2': '管理',
+    'help': '帮助',
+}
+
+# 描述文本中可识别的限制短语及其标签文案
+_RESTRICTION_PATTERNS = [
+    (r"，需要管理员权限", "管理员"),
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)